infra/aws_bootstrap/variables.tf

```diff
@@ -0,0 +1,4 @@
+variable "environment_name" {
```

Comment:
does this need an environment name? I thought the aws_bootstrap was account-wide and didn't need to be launched "once per environment". 🤔